src/pages/docs/data-folder.mdx

```diff
@@ -0,0 +1,129 @@
+    conversational AI,
+    no-subscription fee,
+    large language models,
+    quickstart,
+    getting started,
+    using AI model,
+  ]
+---
+
+import { Tabs } from 'nextra/components'
+import { Callout, Steps } from 'nextra/components'
+
+# Jan Data Folder
+
+After installing Jan, a structured data folder is created on the local filesystem. This setup promotes an organized approach to data management. Below is a guide to understanding how files are organized within Jan data folder.
+
+## Folder Structure
+Jan app data folder should have the following folder structure:
+```yaml
+jan/                                
+  assistants/                           
+  engines/                              
+  logs/                                 
+  models/                               
+  settings/                             
+  threads/                              
+  extensions/                           
+```
+### `jan/` (The Root Directory)
+
+This is the primary directory where all files related to Jan are stored. It typically resides in the user's home directory.
+
+### `assistants/`
+Stores configuration files for various AI assistants. Each assistant within this directory can have different settings.
+
+### `engines/`
+Contains configuration files for different inference engines.
+
+### `logs/`
+Logs from the application are stored here. This is useful for troubleshooting and monitoring the application's behavior over time.
+
+### `models/`
+Stores the AI models that the assistants use to process requests and generate responses.
+
+### `settings/`
+General settings for the application are stored here, separate from individual assistant or engine configurations.
+
+### `threads/`
+Threads history is kept in this directory. Each session or thread is stored in a way that makes it easy to review past interactions.
+
+### `extensions/`
+Extensions enhance Jan's functionality by adding new capabilities or integrating external services.
+
+## Detailed File Layout
+Each of the primary folder structures should have the following detailed structure:
+```yaml
+/assistants
+  /jan                                  
+    assistant.json                      
+  /shakespeare                          
+    assistant.json
+/engines                
+  engine_A.json                         
+/extensions
+  extensions.json                       
+  /@janhq               
+    /extension_A
+     package.json                       
+/logs
+  /app.txt
+/models
+  /model_A
+    model.json                          
+/settings
+  settings.json                         
+  /@janhq               
+    /extension_A_Settings
+     settings.json 
+/threads
+  /jan_thread_A
+   messages.jsonl                       
+   thread.json                          
+  messages.jsonl                        
+```
+
+### Assistants
+- **Default Assistant**: Located in `/assistants/jan/`, it includes an `assistant.json` configuring the default settings and capabilities.
```

Comment:
I expected a sample assistant.json block as follows: 

```json
{
  "avatar": "" // you can change the assistant image here...
  
}
```

Reply:
The idea is to explain what each of the variable means. Maybe this is better done in **table format**?

Please ping us if you are unsure of the intention of a specific property!